tests/DependencyInjection/Compiler/AdminSearchCompilerPassTest.php

```diff
@@ -26,15 +28,24 @@ final class AdminSearchCompilerPassTest extends AbstractCompilerPassTestCase
 {
     public function testProcess(): void
     {
-        $adminFooDefinition = new Definition(PostAdmin::class);
+        $adminFooDefinition = new Definition(
+            PostAdmin::class,
+            ['admin_foo_code', Post::class, CRUDController::class]
+        );
         $adminFooDefinition->addTag('sonata.admin', ['global_search' => true]);
         $this->setDefinition('admin.foo', $adminFooDefinition);
 
-        $adminBarDefinition = new Definition(PostAdmin::class);
+        $adminBarDefinition = new Definition(
+            PostAdmin::class,
+            ['admin_bar_code', Post::class, CRUDController::class]
+        );
         $adminBarDefinition->addTag('sonata.admin', ['global_search' => false]);
         $this->setDefinition('admin.bar', $adminBarDefinition);
 
-        $adminBazDefinition = new Definition(PostAdmin::class);
+        $adminBazDefinition = new Definition(
+            PostAdmin::class,
+            ['admin_bar_code', Post::class, CRUDController::class]
```

Comment:
what would be the use case for someone to use the same code for two admins?

Reply:
My mistake, it need to be 'admin_baz_code'

Reply:
Fixed